### DISCUSSION TITLE
Dynamic pop-up text

### DISCUSSION THREAD

**Discussion by AniAve** [File-New-Project/EarTrumpet#1332] [Ideas]
When you hover a mouse over an icon and scroll you change a volume, right? But the text that pop-ups while yoiu do it doesnt change after it's showing. Can you fix it please?

**Comment by riverar** [comment]
Hi @AniAve sounds like this is related to https://github.com/File-New-Project/EarTrumpet/issues/1298? Can you provide more details in that thread like what OS and EarTrumpet version you're on? Thanks!

**Comment by AniAve** [comment]
Hi! Sorry for the sooo late reply, nevertheless issue is still present.

Versions info:
Windows 10 Home
Ver 22H219045.2728
Windows Feature Experience Pack 120.2212.4190.0
EarTrumpet 2.2.2.0

**Reply by riverar** [reply]
Thanks @AniAve I'll create an issue and try this out asap.
